source/cloud/nvidia/brev.md

```diff
@@ -15,31 +15,39 @@ There are two options to get you up and running with RAPIDS in a few steps, than
 
 ### Option 1. Setting up your Brev GPU Instance
 
-1. Navigate to the [Brev bash](https://brev.nvidia.com/org) and click on "Create your first instance".
+1. Navigate to the [Brev bash](https://brev.nvidia.com/org) and click on "Create new instance".
 
 ![Screenshot of the "Create your first instance" UI](/_static/images/platforms/brev/brev1.png)
 
-2. Select "Container Mode".
+2. Choose a compute type.
 
-![Screenshot showing "Container Mode" highlighted](/_static/images/platforms/brev/brev2.png)
+```{hint}
+New users commonly choose `L4` GPUs for trying things out.
```

Comment:
Below in the screnshoot that uses brev6.png we show the nvidia-smi of a T4, I'm not sure if we care about consistency or if it's just an example. BUt people could probably get started with a T4 too.